tensorflow/core/framework/kernel_def.proto

```diff
@@ -40,6 +40,12 @@ message KernelDef {
   // priority is 0. The higher the priority the better. By default (i.e. if
   // this is not set), we prefer GPU kernels over CPU.
   int32 priority = 6;
+
+  // Subdevice type of this kernel runs on
```

Comment:
nit. Change to:
`Sub-type of device this kernel runs on.`

Reply:
done